search/search_quality/scoring_model.py

```diff
@@ -0,0 +1,82 @@
+from math import exp, log
+import numpy as np
+import pandas as pd
```

Comment:
fmt looks ugly at first sight, but when you get one's hand in writing Go code, you start to appreciate such useful shortcuts.